src/Filter.scss

```diff
@@ -28,7 +28,7 @@ nav li {
   }
 }
 
-.population-filter {
+.filter-class {
   position: absolute;
```

Comment:
I think we should rename filter-class something else? Since its a class it doesnt need to be named class